module/utils.mjs

```diff
@@ -469,7 +490,8 @@ export function registerHandlebarsHelpers() {
     "dnd5e-linkForUuid": (uuid, options) => linkForUuid(uuid, options.hash),
     "dnd5e-itemContext": itemContext,
     "dnd5e-numberFormat": (context, options) => formatNumber(context, options.hash),
-    "dnd5e-textFormat": formatText
+    "dnd5e-textFormat": formatText,
+    "dnd5e-formatModifier": formatModifier
```

Comment:
Would be nice to move this up so these are in alphabetical order (though I suppose `itemContext` is also in the wrong spot :/).